src/app/components/reports-time-graph/reports-time-graph.component.ts

```diff
@@ -62,7 +62,7 @@ export class ReportsTimeGraphComponent implements OnInit {
   selectedView: string = null;
 
   /** Viewport base width of the line chart */
-  w: number = Math.max(document.documentElement.clientWidth, window.innerWidth || 0);
+  w: number = Math.max(document.documentElement.clientWidth, window.innerWidth);
```

Comment:
Why was this default removed?

Reply:
I was testing something and forgot to put it back. I will handle that.

Reply:
I was going through the files again, and I just remembered why I removed the default. The documentation for window states that window.innerWidth does not have a default value. I assume this means that window.innerWidth can be null or some other falsy. However, in the case that it is falsy, in this context, Math.max(any number, falsy) will treat the falsy as 0. So there is no point in using the guard. Unless I misread this.